software/kaneo.yml

```
@@ -0,0 +1,14 @@
+name: "Kaneo"
+website_url: "https://kaneo.app/"
+source_code_url: "https://github.com/usekaneo/kaneo"
+description: "A project management platform focused on simplicity and efficiency."
+licenses:
+  - MIT
+platforms:
+  - Nodejs
+  - K8S
+  - Docker
+tags:
+  - Ticketing
+  - Task Management & To-do Lists
```

Since it calls itself a Project Management Platform this seems to be more fitting
```suggestion
tags:
  - Software Development - Project Management
```